docs/feel-expression.md

```diff
@@ -158,7 +158,7 @@ true and false
 // false
 
 true and null        
-// false
+// null
```

Comment:
Did this change because of your change? Or was this wrong in the documentation?

Reply:
Nope. The documentation was a lie :sweat_smile: